jetstream/core/server_lib.py

```diff
@@ -148,6 +152,13 @@ def run(
   logging.info("Starting server on port %d with %d threads", port, threads)
 
   jetstream_server.start()
+
+  # Setup Jax Profiler
+  if enable_jax_profiler:
+    logging.info("Starting JAX profiler server on port 9999")
+    jax.profiler.start_server(9999)
```

Comment:
Instead of hardcode port number, can you define a profile port flag and use it?

Reply:
https://jax.readthedocs.io/en/latest/profiling.html#manual-capture-via-tensorboard, I think the tensorboard is connecting to port 9999 to the jax profiler?

Reply:
Updated.